source/features/bugs-tab.tsx

```diff
@@ -12,7 +12,7 @@ import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 import {highlightTab, unhighlightTab} from '../helpers/dom-utils';
 
-const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|:\w+:bug)$/i;
+const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|(:[\w_-]+:|\p{Emoji})\s?bug)$/iu;
```

Comment:
```suggestion
const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|(:[\w-]+:|\p{Emoji})bug)$/iu;
```


 * `_` is already included in `\w`
 * we don't need to acount for a space, they're removed before testing